site/weave-docker-api.md

```diff
@@ -10,6 +10,21 @@ network when they are started using the ordinary Docker
 or the [remote API](https://docs.docker.com/reference/api/docker_remote_api/),
 instead of `weave run`.
 
+###Attaching Containers to a Weave Network
+
+There are two ways to attach containers to a Weave network, 
+the first uses the Weave Net Docker API Proxy, and is described in this section.  
+The other method uses the Docker Network Plugin framework. It’s your choice which method to use. 
+
+To use the Docker Network Plugin, launch Weave with the --net flag.  Note that after 
+you’ve launched Weave Net with this flag, then the Weave Docker API Proxy is automatically disabled.  
```

Comment:
The `--net` flag gets specified in `docker run`, i.e. when starting containers. Not when launching weave.

Also, specifying `--net` does *not* "use the Docker Network Plugin". It disables automatic attachment to the Weave network by the proxy. Nothing more. What happens networking-wise to such a container is completely up to Docker and what exactly you specified in `--net`.

Finally, there is another option that has the same effect: `-e WEAVE_CIDR=none`. It's mentioned in a few places in the docs. I reckon that option and the explanation about `--net` should be consolidated into one place.